add hero img, fixed pad, center txt, chg color

hmcoombs opened the pull request:
What do you think

hmcoombs committed:
add hero img, fixed pad, center txt, chg color

Ksappie commented:
is it just a blue background with black text? just making sure I cloned the right file

hmcoombs commented:
Hey Kyle, I am going to work/look at it again tomorrow to make sure I added all the requirements and did the pushes right.  I will also check on what you pushed.

Thanks,
Mike

________________________________
From: Ksappie ***@***.***>
Sent: Sunday, July 17, 2022 4:38 PM
To: Ksappie/collab-practice-KS ***@***.***>
Cc: smackmhard ***@***.***>; State change ***@***.***>
Subject: Re: [Ksappie/collab-practice-KS] add hero img, fixed pad, center txt, chg color (PR #5)


is it just a blue background with black text? just making sure I cloned the right file

—
Reply to this email directly, view it on GitHub<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FKsappie%2Fcollab-practice-KS%2Fpull%2F5%23issuecomment-1186611638&data=05%7C01%7C%7C4daf73c62b684b2f104508da683ca272%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637936906877383450%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=01lU2mCD8e5W5bDq%2BB8FS2q0iyyywa1ryu0Ms2uJT3Q%3D&reserved=0>, or unsubscribe<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAZACZQRTUPVSBEFXLZCGYZLVUR4L3ANCNFSM53YZVIHQ&data=05%7C01%7C%7C4daf73c62b684b2f104508da683ca272%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637936906877383450%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=xxUdHrTR0NbPBuHqMIKLs6EF0hYOFMfVCqsQH57RAQU%3D&reserved=0>.
You are receiving this because you modified the open/close state.Message ID: ***@***.***>

hmcoombs commented:
Hey Kyle,  The hero image is a light blue background with black text that says:

this is my hero image/jumbotron
This will bring attention to my page

There is padding on the top and bottom of the hero.
________________________________
From: Ksappie ***@***.***>
Sent: Sunday, July 17, 2022 4:38 PM
To: Ksappie/collab-practice-KS ***@***.***>
Cc: smackmhard ***@***.***>; State change ***@***.***>
Subject: Re: [Ksappie/collab-practice-KS] add hero img, fixed pad, center txt, chg color (PR #5)


is it just a blue background with black text? just making sure I cloned the right file

—
Reply to this email directly, view it on GitHub<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FKsappie%2Fcollab-practice-KS%2Fpull%2F5%23issuecomment-1186611638&data=05%7C01%7C%7C4daf73c62b684b2f104508da683ca272%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637936906877383450%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=01lU2mCD8e5W5bDq%2BB8FS2q0iyyywa1ryu0Ms2uJT3Q%3D&reserved=0>, or unsubscribe<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAZACZQRTUPVSBEFXLZCGYZLVUR4L3ANCNFSM53YZVIHQ&data=05%7C01%7C%7C4daf73c62b684b2f104508da683ca272%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637936906877383450%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=xxUdHrTR0NbPBuHqMIKLs6EF0hYOFMfVCqsQH57RAQU%3D&reserved=0>.
You are receiving this because you modified the open/close state.Message ID: ***@***.***>
